### PR TITLE
PAYEE-423: Upgraded to new FE library

### PR DESCRIPTION
**Pull request by Jamie22Butler**
- [x] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: app/views/benefits/RemoveCompanyBenefitCheckYourAnswersView.scala.html [hunk 1]
```diff
@@ -22,37 +22,50 @@
...
+    backLinkId = "backLink"
 ){
 
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        @header
+    </div>
+</div>
+
+<div class="govuk-grid-row">
```
Comment (sergi-hmrc) [inline app/views/benefits/RemoveCompanyBenefitCheckYourAnswersView.scala.html:51]: these two are redundant

Comment (Jamie22Butler) [reply app/views/benefits/RemoveCompanyBenefitCheckYourAnswersView.scala.html:51]: Screen Shot 2022-07-15 at 14 27 51

Removing them does this :/

**Review by sergi-hmrc — APPROVED** [review]

File: app/views/benefits/RemoveCompanyBenefitCheckYourAnswersView.scala.html (hunk 1 above)
Comment (Jamie22Butler) [reply app/views/benefits/RemoveCompanyBenefitCheckYourAnswersView.scala.html:51]: Takes up the whole screen, whereas adding them aligns them to that of the prototype.

Comment (sergi-hmrc) [reply app/views/benefits/RemoveCompanyBenefitCheckYourAnswersView.scala.html:51]: i mean u have: 
grid-row
grid-2/3

grid-row
grid-2/3

**Review by vivekhmrc — APPROVED** [review]
LGTM 👍

**Pull request merged**